Require/index.js

```diff
@@ -209,6 +209,6 @@ describe("TNS require", function () {
 
     it("should handle JSON file errors", function () {
         require('./RequireJsonCorruptFile');
-        expect(TNSGetOutput()).toBe("File: 'file:///app/shared/Require/RequireJsonCorruptFile/test.json'. JSON Parse error: Unterminated string");
+        expect(TNSGetOutput()).toBe("There was a corrupt or invalid package.json.");
```

Comment:
Something like `.toMatch(/JSON Parse error: Unterminated string$|Uncaught SyntaxError: Unexpected token ILLEGAL$/)` looks good to me. Would be glad to hear other opinions on this.